spec/basket_spec.rb

```diff
@@ -2,19 +2,35 @@
 
 RSpec.describe Basket do
 
-  it "creates basket" do
-    expect Basket.new.is_a? Basket
-  end
+  let(:product1) { Product.new(name: "Football", price: 79) }
```

Comment:
Will change for next commit. Just wanted to know why you suggested this before changing. You clarified this already but it was after current commit.